tests/acceptance/features/apiSpaces/uploadSpaces.feature

```diff
@@ -0,0 +1,50 @@
+@api @skipOnOcV10
+Feature: Upload files into a space
+  As a user
+  I want to be able to work with project spaces and quota
+
+  Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+  See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Bob" has been created with default attributes and without skeleton files
```

Comment:
user "Bob" does not mentioned in the test

Reply:
added more scenarios